.travis.yml

```diff
@@ -38,6 +34,9 @@ matrix:
         - os: osx
           language: generic
           env: TERRYFY_PYTHON='macpython 3.7.0'
+        - os: osx
+          language: generic
+          env: TERRYFY_PYTHON='macpython 3.8.0a3'
```

Comment:
This [doesn't appear to work](https://travis-ci.org/zopefoundation/persistent/jobs/523996701)